website/components/pages/home.tsx

```diff
@@ -38,128 +38,130 @@ export default function Home() {
     toast.success("Copied to clipboard");
   };
 
-  return (
-    <>
-      <Head>
-        <title>CloudQuery</title>
-        <meta
-          name="og:description"
-          content="CloudQuery is an open source high performance data integration platform built for developers."
-        />
-      </Head>
+  return <>
```

Comment:
Changes here are due to running https://nextjs.org/docs/advanced-features/codemods#new-link